realm/realm-library/src/objectServer/java/io/realm/SyncUser.java

```diff
@@ -274,31 +274,7 @@ public void logout() {
             // the similar SyncConfiguration using the same identity, but with different (new)
             // refresh-token.
             realms.clear();
-
-            // Finally revoke server token. The local user is logged out in any case.
-            final AuthenticationServer server = SyncManager.getAuthServer();
-            // don't reference directly the refreshToken inside the revoke request
-            // as it may revoke the newly acquired and refresh_token
-            final Token refreshTokenToBeRevoked = refreshToken;
-
-            ThreadPoolExecutor networkPoolExecutor = SyncManager.NETWORK_POOL_EXECUTOR;
-            networkPoolExecutor.submit(new ExponentialBackoffTask<LogoutResponse>() {
-
-                @Override
-                protected LogoutResponse execute() {
-                    return server.logout(refreshTokenToBeRevoked, getAuthenticationUrl());
-                }
-
-                @Override
-                protected void onSuccess(LogoutResponse response) {
-                    SyncManager.notifyUserLoggedOut(SyncUser.this);
-                }
-
-                @Override
-                protected void onError(LogoutResponse response) {
-                    RealmLog.error("Failed to log user out.\n" + response.getError().toString());
-                }
-            });
+            SyncManager.notifyUserLoggedOut(SyncUser.this);
```

Comment:
Is this related with https://github.com/realm/ros/issues/261 ?
no need to send logout request anymore?

Reply:
Yup, it is related to that issue. Until that is resolved in some way sending the logout request isn't supported.